contributing/code/standards.rst

```diff
@@ -167,6 +167,29 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-comparisons:
+
+Comparisons
+-----------
+
+Always use `identical comparison`_ unless you need type juggling:
+
+instead of:
+
+.. code-block:: php
+
+    if (1 == $integerExpected) {
+        // ...
+    }
+
+it should be:
+
+.. code-block:: php
+
+    if (1 === $integerExpected) {
+        // ...
+    }
```

Comment:
What about using one code block for this to make it more compact?

```rst
Always use `identical comparison`_ unless you need type juggling::

    // use
    if (1 === $integerExcepted) {
        // ...
    }

    // instead of
    if (1 == $integerExcepted) {
        // ...
    }
```